src/App.js

```diff
@@ -191,25 +190,21 @@ class App extends Component {
         <section id="main" className="container">
           {this.state.articles
             .filter(article => {
-              if (this.state.selectedSource === "All") {
-                return article;
-              } else if (article.source === this.state.selectedSource) {
-                return article;
-              }
-              return null;
+              return this.state.selectedSource === "All"
+                || article.source === this.state.selectedSource
             })
             .filter(article => {
               return article.title
                 .toLowerCase()
-                .includes(this.state.searchText);
+                .includes(this.state.searchText.toLowerCase());
             })
             .sort((a, b) => {
               return b.date - a.date;
-            })
+            }) // Fantastic use of filtering, sorting + mapping!
             .map(article => {
               return (
                 <Article
-                  key={`${article.source}${article.id}`}
+                  key={`${article.source}${article.id}` /* Solid key creation - this should always be unique */}
```

Comment:
May seem like a simple thing, but this shows a really strong understanding of your data 👍 👍